README.md

```diff
@@ -57,6 +57,8 @@ Only once you need to run `go run ./cmd/appsctl/ provision bucket` to create the
 
 To provision an app run `go run ./cmd/appsctl/ provision app /PATH/TO/YOUR/APP/BUNDLE`. Use `--update` to update the lambda functions if they already exist. This option should not be used in production.
 
+Run `go run ./cmd/appsctl/ generate-terraform-data /PATH/TO/YOUR/APP/BUNDLE` to generate terraform variables. It will print the indented json directly in the console. Pipe as needed.
```

Comment:
We should make clear who is to invoke this script and in what context (e.g. the cloud team pipeline, etc.). That will also give info on decisions like to `createAWSClient**WithoutCredentials**)